README.md

```diff
@@ -16,10 +16,10 @@ Windows, macOS, Linux, iOS and Android. Built for you, maintained for Tauri.
 
 Tao provides the following features, which can be enabled in your `Cargo.toml` file:
 * `serde`: Enables serialization/deserialization of certain types with [Serde](https://crates.io/crates/serde).
-* `menu`: Enables system tray and more menu item variants on **Linux**. This flag is enabled by default.
-  You can still create those types if you disable it. They just don't create the actual objects. We set this flag
-  because some implementations require more installed packages.  Disable this if you don't want to install those
-  additional packages.
+* `tray`: Enables system tray and more menu item variants on **Linux**. This flag is enabled by default.
+  You can still create those types if you disable it. They just don't create the actual objects. We set this flag because some implementations require more installed packages. Disable this if you don't want to install those additional packages.
```

Comment:
```suggestion
  You can still create those types if you disable it. They just don't create the actual objects. We set this flag because some implementations require more installed packages. Disable this if you don't want to install `libappindicator` package.
```